content/pages/profile/index.md

```diff
@@ -25,7 +25,7 @@ entryname: profile-360
 
 <script>
   (function() {
-    var accountHomeLink = document.getElementById('account-home');
+    var accountHomeLink = document.getElementById('home');
     accountHomeLink.addEventListener('click', function(ev) {
```

Comment:
I'm looking forward to removing this code once we have the React breadcrumbs in place.